WebRoot/js/zimbraMail/share/view/ZmAutocompleteListView.js

```diff
@@ -1180,15 +1180,7 @@ function(list, context) {
 			row.index = i;
 			var html = [], idx = 0;
 			var cell = row.insertCell(-1);
-			cell.className = "AutocompleteMatchIcon";
-			if (match.icon) {
-				cell.innerHTML = (match.icon.indexOf('Dwt') !== -1) ? ["<div class='", match.icon, "'></div>"].join("") :
-								 									 AjxImg.getImageHtml(match.icon);
-			} else {
-				cell.innerHTML = "&nbsp;";
-			}
-			cell = row.insertCell(-1);
-			cell.innerHTML = match.text || "&nbsp;";
+			cell.innerHTML = ( "<span class='acRowTop'>" + match.name + "</span>" + "<br>" + match.email) || "&nbsp;";
```

Comment:
minor: Though `br` is self closing tag, for readability point of view, how if we close it i.e `<br/>`